### PR TITLE
Challenge6

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: test/6.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const callX2= require('../solutions/6.js');
+
+const testing = (num)=>{
+  let count = 0;
```
Comment (songz) [inline test/6.js]: let compare = true;

File: test/6.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  const hello = (a)=>{
+    console.log(a);
```
Comment (songz) [inline test/6.js]: `if(count !== a) compare = false;`

File: test/6.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    count+=1;
+  }
+  callX2(hello,num);
+  if(count==num){
```
Comment (songz) [inline test/6.js]: `if(count==num && compare)`

File: test/6.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+const callX2= require('../solutions/6.js');
+
+const testing = (num)=>{
+  let count = 0; 
+  let passing = true;
+
+  const hello = (a)=>{
+    console.log(a);
```
Comment (bramakrishna16) [inline test/6.js]: you can remove console.log

**Review by bramakrishna16 — APPROVED** [review]

File: test/6.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+const callX2= require('../solutions/6.js');
+
+const testing = (num)=>{
+  let count = 0; 
+  let passing = true;
+
+  const hello = (a)=>{
+    count+=1;
+    if(a!=count){
```
Comment (roycep) [inline test/6.js]: You can shorten your tests by removing the whole block on lines 9-11

`if(a!=count){...`

Then shorten line 14 to check "if the count is equal to num"

File: solutions/6.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+const callX2=(func,num,i=1)=>{
+  func(i);
```
Comment (roycep) [inline solutions/6.js]: Double check the value you are passing into func.

i is the number of calls you have made, but that is not the value the function should use

**Pull request merged**